python/pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "sigstore-protobuf-specs"
+version = "0.0.1"
+description = "A library for serializing and deserializing Sigstore messages"
+readme = "../README.md"
+license = { file = "LICENSE" }
+authors = [
+  { name = "Sigstore Authors", email = "sigstore-dev@googlegroups.com" }
+]
+classifiers = [
+  "License :: OSI Approved :: Apache Software License",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Development Status :: 4 - Beta",
+  "Intended Audience :: Developers",
+  "Topic :: Security",
+  "Topic :: Security :: Cryptography",
+]
+dependencies = [
+  "betterproto",
+]
+requires-python = ">=3.7"
+
+[project.urls]
+Homepage = "https://pypi.org/project/sigstore-protobuf-specs/"
+Issues = "https://github.com/sigstore/protobuf-specs/issues"
+Source = "https://github.com/sigstore/protobuf-specs"
+
+[project.optional-dependencies]
+dev = [
+  "betterproto[compiler]",
+  "grpcio-tools",
```

Comment:
To get things working, I'm listing `grpcio-tools` as a dependency and using the bundled `protoc` instead of invoking the Docker container.

Before this gets merged in, I can fiddle with this to plug `betterproto` into the Docker container's `protoc`. Or we can avoid the Docker container altogether and use this `protoc` for every language.